src/components/Toolbar/style.css

```diff
@@ -5,3 +5,9 @@
 .toolbar {
   width: 100%;
 }
+
+@media only screen and (max-width : 768px){
+  .toolbar :global(.toolbar-pf-action-right){
+    float: right
```

Comment:
Did you try to put this class outside `@media` ?

Reply:
You're right. I don't think it would make a difference. Nice catch!